agent/lib/kontena/service_pods/creator.rb

```diff
@@ -153,7 +153,7 @@ def volumes_exist?(service_pod)
 
       # @param [Docker::Container] container
       def cleanup_container(container)
-        container.stop('timeout' => 10)
+        container.stop('timeout' => container.stop_grace_period)
```

Comment:
Would it be better to use the `@service_pod.stop_grace_period` here, so changes to the service `stop_grace_period` would already apply on the immediately following deploy?